gpu/kinfu_large_scale/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ target_link_libraries(${LIB_NAME} pcl_cuda pcl_common pcl_io pcl_gpu_utils pcl_g
 
 target_compile_options(${LIB_NAME} PRIVATE $<$<COMPILE_LANGUAGE:CUDA>:--ftz=true;--prec-div=false;--prec-sqrt=false>)
 
-PCL_MAKE_PKGCONFIG(${LIB_NAME} COMPONENT ${SUBSYS_NAME} DESC ${SUBSYS_DESC} PCL_DEPS ${SUBSYS_DEPS} EXT_DEPS ${EXT_DEPS})
+PCL_MAKE_PKGCONFIG(${LIB_NAME} COMPONENT ${SUBSYS_NAME} DESC ${SUBSYS_DESC} PCL_DEPS ${SUBSYS_DEPS} EXT_DEPS "")
```

Comment:
Doesn't this change bite you when the next dependency is added to EXT_DEPS?

Reply:
Fair point, but I think it is rather unlikely that anything else is added to `EXT_DEPS`, since kinfu_large_scale appears to be rarely used and is not compatible with CUDA 12 in its current state.
And besides, what would be the alternative? Even if we put `vtk` directly into `PCL_SUBSYS_DEPEND`, someone might forget to double-add a potential new external dependency.